IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationTokenRequest.m

```diff
@@ -63,7 +63,7 @@ + (BOOL)fillRequest:(MSIDBrokerOperationTokenRequest *)request
     request.nonce = parameters.nonce;
     request.clientSku = parameters.clientSku;
     request.skipValidateResultAccount = parameters.skipValidateResultAccount;
-        
+    request.forceRefresh = parameters.forceRefresh;
```

Comment:
Do we also need to pass it in legacy broker request?

Reply:
I think we don't as it for silent operation only so only SSO-Ext flow.

Reply:
I thought there's also scenario for legacy broker, when we might have a valid access token and we ignore forceRefresh parameter?

Reply:
Discussed offline, we leave it for now as don't want change in public API